<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QMainWindow" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Othello Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>650</width>
      <height>660</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="buttonDemoGame">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Autori</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNewGame">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Nová Hra</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLoadGame">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Načítať Hru</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QComboBox" name="comboBoxDeskSize">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPlayer1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Vytvoriť hru</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxAlgorithm">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Späť do menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Veľkosť hracej dosky</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>66</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hráč 2:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPlayer2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Algoritmus</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>66</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hráč 1:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackToGame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Späť na hru</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>640</width>
        <height>550</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonNewGame2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Nová hra</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMove">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Na ťahu:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLoad2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>81</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Načítať</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSave">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>81</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Uložiť</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonChangeGame">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>480</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Zmeniť hru</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPlayer1Score">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>310</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPlayer2Score">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>370</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonPass">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Pass</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxGame">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>440</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="buttonRedo">
      <property name="geometry">
       <rect>
        <x>565</x>
        <y>70</y>
        <width>55</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::RightArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="buttonUndo">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>70</y>
        <width>55</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::LeftArrow</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelStatus">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>9</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelMove_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Skóre</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
